addons/crm/models/crm_lead.py

```diff
@@ -1307,6 +1323,60 @@ def _handle_salesmen_assignment(self, user_ids=False, team_id=False):
     # CLASSIFICATION TOOLS
     # --------------------------------------------------
 
+    @api.depends('email_from', 'partner_id', 'contact_name', 'partner_name')
+    def _compute_potential_lead_duplicates(self):
```

Comment:
As this is a compute method it should be located with other compute (should be ordered following fields definition, but not always doable if original file is not already ordered).